code/game/machinery/computer/timeclock_vr.dm

```diff
@@ -208,8 +208,8 @@
 /obj/machinery/computer/timeclock/proc/checkCardCooldown()
 	if(!card)
 		return FALSE
-	if((world.time - card.last_job_switch) < 15 MINUTES)
-		to_chat(usr, "You need to wait at least 15 minutes after last duty switch.")
+	if((world.time - card.last_job_switch) < 10 MINUTES)
+		to_chat(usr, "You need to wait at least 10 minutes after last duty switch.")
```

Comment:
Would be great to have a way to make this display the time you have **before** it allows you to switch back on or off duty. You have... 250 seconds left before you can change your access permissions again" or whatever.

Reply:
I'd prefer to have this in a separate PR.